Source/Processors/SourceNode/SourceNode.h

```diff
@@ -61,6 +61,9 @@ class PLUGIN_API SourceNode : public GenericProcessor
 
     /* Broadcasts a message from the DataThread to all other processors*/
     void broadcastDataThreadMessage(String msg);
+    
+    /* Broadcasts a message from the DataThread to all other processors*/
+    void sendDataThreadConfigMessage(GenericProcessor* destProcessor, String msg);

```

Comment:
why have this alias of `GenericProcessor::sendConfigMessage`? Seems more straightforward to have callers call `sendConfigMessage` directly

Reply:
I made this wrapper for cases when `DataThread` plugins needed to send a config message, as the Thread's `SourceNode` is what actually derives from `GenericProcessor`. `sendBroadcastMessage` and `sendConfigMessage` are both protected members of `GenericProcessor`, but, after looking at this again, these `SourceNode` wrappers do defeat the purpose of keeping these members proctected.

I believe `MessageCenter` has a lot `Thread` safety checks, so careless use of message passing on non-`ProcessorGraph` threads can lead to unintended consequences. If this we want to enforce this access limitation, which seems reasonable given the intention of cross-processor communication, then perhaps `DataThread` should be a friend of `SourceNode`? Or maybe there is a better way to  define this relationship?